rest-api-spec/src/main/resources/rest-api-spec/api/cluster.delete_weighted_routing.json

```diff
@@ -1,19 +1,31 @@
 {
   "cluster.delete_weighted_routing": {
     "documentation": {
-      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/weighted-routing/delete",
+      "url": "https://opensearch.org/docs/2.5/api-reference/cluster-api/cluster-awareness/",
```

Comment:
Why not point to `latest`? That seems to be the standard. Also the `2.5` reference will go stale.

```
% git grep 'opensearch.org/docs/' rest-api-spec/src/main/resources/rest-api-spec/api/
rest-api-spec/src/main/resources/rest-api-spec/api/cat.cluster_manager.json:      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-master/",
rest-api-spec/src/main/resources/rest-api-spec/api/cluster.delete_decommission_awareness.json:      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/decommission/",
rest-api-spec/src/main/resources/rest-api-spec/api/cluster.delete_weighted_routing.json:      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/weighted-routing/delete",
rest-api-spec/src/main/resources/rest-api-spec/api/cluster.get_decommission_awareness.json:      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/decommission/",
rest-api-spec/src/main/resources/rest-api-spec/api/cluster.get_weighted_routing.json:      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/weighted-routing/get",
rest-api-spec/src/main/resources/rest-api-spec/api/cluster.put_decommission_awareness.json:      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/decommission/",
rest-api-spec/src/main/resources/rest-api-spec/api/cluster.put_weighted_routing.json:      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/weighted-routing/put",
rest-api-spec/src/main/resources/rest-api-spec/api/create_pit.json:      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/point_in_time/",
rest-api-spec/src/main/resources/rest-api-spec/api/delete_all_pits.json:      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/point_in_time/",
rest-api-spec/src/main/resources/rest-api-spec/api/delete_pit.json:      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/point_in_time/",
rest-api-spec/src/main/resources/rest-api-spec/api/get_all_pits.json:      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/point_in_time/",
rest-api-spec/src/main/resources/rest-api-spec/api/remote_store.restore.json:      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/remote-store#restore",
```